tests/test_backends.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+"""
+Tests for the `openedx-cas` backends module.
+"""
+import urllib.parse
+from unittest.mock import Mock, patch
+
+from ddt import data, ddt, unpack
+from django.conf import settings
+from django.test import TestCase, override_settings
+from social_core.exceptions import AuthFailed, AuthMissingParameter
+
+from openedx_cas.backends import CASAuth
+
+
+@ddt
+class TestOpenEdxCasAuthBackend(TestCase):
+    """
+    Test class to verify standard behavior of utility methods that belong to CASAuth.
+    """
+
+    def setUp(self) -> None:
+        super().setUp()
+        strategy = Mock()
+        self.cas_backend = CASAuth(strategy=strategy)
+
+    def test_cas_service_url_defined_in_settings(self):
+        """
+        This method is used to verify the CAS_SERVICE_URL variable in settings is defined
+        Expected behavior:
+        CASAuth.auth_url includes a query param called 'service' which value is a parsed string with the CAS_SERVICE_URL
+        """
+        auth_url = self.cas_backend.auth_url()
+        self.assertIn(urllib.parse.quote(settings.CAS_SERVICE_URL), auth_url)
```

Comment:
Remember triple A testing:
```suggestion
        auth_url = self.cas_backend.auth_url()
        
        self.assertIn(urllib.parse.quote(settings.CAS_SERVICE_URL), auth_url)
```
Repeat in the other tests